components/drata/sources/terminated-employee/terminated-employee.mjs

```diff
@@ -7,7 +7,7 @@ const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/
 export default {
   key: "drata-terminated-employee",
   name: "Employee Terminated",
-  description: `Emit new event when an employee is terminated. [See docs here](${docsLink}).`,
+  description: `Emit a new event when an employee is terminated. [See docs here](${docsLink}).`,
```

Comment:
```suggestion
  description: `Emit a new event when an employee is terminated. [See the documentation](${docsLink}).`,
```

Reply:
Changed "docs here" to "the documentation" to follow Google's Developer docs style guide.